### DISCUSSION TITLE
有人盗用说是他们开发的然后商业卖卖，我说要标记开源地址然后被拉黑

### DISCUSSION THREAD

**Discussion by myh66** [binary-husky/gpt_academic#1176] [General]
我留言了一句要标注开源地址就被拉黑了，下面是哔哩哔哩主页，理性讨论
 他说是他们做的
@binary-husky 
https://space.bilibili.com/34221342
截图
![image](https://github.com/binary-husky/gpt_academic/assets/53999290/9dc2cf64-d818-41f2-9297-31e2aafdf8f5)
![image](https://github.com/binary-husky/gpt_academic/assets/53999290/6490ff8f-583e-4102-bfb9-45359e1a7643)
![image](https://github.com/binary-husky/gpt_academic/assets/53999290/84d8d08d-2b9d-4aed-a5ed-5fbadb9a46a7)

**Comment by binary-husky** [comment]
离谱

**Comment by dynamor2019** [comment]
我也发现了，留言了这是个开源项目

**Comment by liujiaxi1999** [comment]
是的，他好像还做了阉割...

**Comment by laundry2** [comment]
我给举报了，看看能不能给整下架

**Comment by AI2Inventor** [comment]
这个up主是github这个团队的？

**Reply by binary-husky** [reply]
不是，我们有以下两个官方的b站账户：

我们的B站开发演示视频：https://space.bilibili.com/475312678
我们的B站营收演示视频：https://space.bilibili.com/3493270682339966

**Comment by Nextstrain9000** [comment]
保护我方二进制哈士奇
